src/server/scripts/EasternKingdoms/ZulAman/zulaman.h

```diff
@@ -25,19 +25,20 @@
 
 enum DataTypes
 {
-    DATA_NALORAKK                       = 0,
-    DATA_AKILZON                        = 1,
-    DATA_JANALAI                        = 2,
-    DATA_HALAZZI                        = 3,
-    DATA_HEXLORD                        = 4,
-    DATA_ZULJIN                         = 5,
-    MAX_ENCOUNTER                       = 6,
-    DATA_SPIRIT_LYNX                    = 7,
-    TYPE_RAND_VENDOR_1                  = 8,
-    TYPE_RAND_VENDOR_2                  = 9,
-    DATA_STRANGE_GONG                   = 10,
-    DATA_MASSIVE_GATE                   = 11,
-    DATA_HEXLORD_GATE                   = 12
+    DATA_HARRISON_JONES                 = 0,
```

Comment:
Please add it to the bottom of the list. This will affect the encounter counter and consequently corrupt the instance save string.